task_processing/interfaces/task_executor.py

```diff
@@ -17,7 +17,16 @@
 
 def make_task_config(image="ubuntu:xenial", cmd="/bin/true", cpus=0.1,
```

Comment:
I'm concerned about this, let's add a TODO to use attrs or record classes or something?